Algebraic_kernel_d/examples/Algebraic_kernel_d/CMakeLists.txt

```diff
@@ -4,15 +4,13 @@ project(Algebraic_kernel_d_Examples)
 find_package(CGAL REQUIRED COMPONENTS Core)
 
 find_package(MPFI QUIET)
-
 if(MPFI_FOUND AND NOT CGAL_DISABLE_GMP)
   include(${MPFI_USE_FILE})
-  include(CGAL_VersionUtils)
   create_single_source_cgal_program("Compare_1.cpp")
   create_single_source_cgal_program("Construct_algebraic_real_1.cpp")
   create_single_source_cgal_program("Isolate_1.cpp")
   create_single_source_cgal_program("Sign_at_1.cpp")
   create_single_source_cgal_program("Solve_1.cpp")
 else()
-    message(STATUS "This program requires the CGAL, CGAL_Core and MPFI libraries, and will not be compiled.")
+  message("NOTICE: This project requires the MPFI library and GMP support, and will not be compiled.")
```

Comment:
According to the CMake documentation (https://cmake.org/cmake/help/latest/command/message.html):

>(none) or NOTICE
>
>    Important message printed to stderr to attract user's attention.

So isn't this a bit double?

Reply:
`NOTICE` is not yet available in the oldest CMake version still supported, hence no `NOTICE`. I have put "NOTICE:" like this so this is easier to update these "none" occurrences once the minimal version is upgraded.

The other point is that it prints something in capital letters, in case your stderr is not showing up in red.

Reply:
Valid argument, in the documentation is says (as reference):
> New in version 3.15: Added the NOTICE, VERBOSE, DEBUG, and TRACE levels.